### PR TITLE
Unique database values

### LINKED ISSUE #76: Prevent duplicate responses in database
#111 
Currently, the database doesn't explicitly prevent duplicate responses.

It's possible to have multiple rows with identical user_id, question_id and response values

### PR DESCRIPTION
**Pull request by minaorangina**
Resolves #76

### CODE REVIEW

**Comment by sohilpandya** [comment]
@minaorangina a few tests are failing and hence circle ci is failing too.

File: server/lib/queries.json [hunk 1]
```diff
@@ -7,5 +7,7 @@
...
-    "getQuizResult": "SELECT COUNT(CASE WHEN responses.response = questions.correct_answer THEN 1 END) FROM responses JOIN questions ON questions.question_id = responses.question_id WHERE responses.user_id = ($1) AND responses.quiz_id = ($2);"
+    "getQuizResult": "SELECT COUNT(CASE WHEN responses.response = questions.correct_answer THEN 1 END) FROM responses JOIN questions ON questions.question_id = responses.question_id WHERE responses.user_id = ($1) AND responses.quiz_id = ($2);",
+    "saveStudentResponse": "INSERT INTO responses (user_id, quiz_id, question_id, response) VALUES ($1, $2, $3, $4) ON CONFLICT (user_id, quiz_id, question_id) DO UPDATE SET response = $4 WHERE (responses.user_id, responses.quiz_id, responses.question_id) = ($1, $2, $3);",
```
Comment (sohilpandya) [inline server/lib/queries.json]: Does this query work? I thought pg module doesn't allow the use of same variable more than once. so $1 being used twice should be an error? 

OR does it work ok?

File: src/scss/bulma.css [hunk 2]
```diff
@@ -5093,12 +5093,12 @@ a.box:active {
   display: -webkit-box;
   display: -webkit-flex;
   display: -ms-flexbox;
-  display: flex;
+  display: inline-flex;
```
Comment (sohilpandya) [inline src/scss/bulma.css]: necessary to turn it to inline-flex?

File: server/lib/queries.json (hunk 1 above)
Comment (minaorangina) [inline server/lib/queries.json]: Works fine. Where did you read that that wasn't possible?

Comment (minaorangina) [inline server/lib/queries.json]: I just ran a quiz a couple of times and had no issues

Comment (sohilpandya) [inline server/lib/queries.json]: my bad I was looking at this. 
https://github.com/brianc/node-postgres/wiki/Prepared-Statements#multi-statement-parameterized-queries

**Pull request closed without merging**